guides/common/modules/con_restricting-hosts-access-to-content.adoc

```diff
@@ -0,0 +1,66 @@
+[id="Restricting_Hosts_Access_to_Content_{context}"]
+= Restricting Hosts' Access to Content
+
+{Project} offers multiple options for restricting host access to content.
+To give hosts access to a specific subset of the content managed by {Project}, you can use the following strategies.
+{Team} recommends to consider implementing the strategies in the order listed here:
+
+Content Views and lifecycle environments::
+Use Content Views and lifecycle environments, incorporating Content View filters as needed.
++
+For more information about Content Views, see xref:Managing_Content_Views_{context}[].
++
+For more information about lifecycle environments, see xref:Creating_an_Application_Life_Cycle_{context}[].
+
+Content overrides::
+By default, content hosted by {Project} can be either enabled or disabled.
+In custom products, repositories are always disabled by default, while Red{nbsp}Hat products can be either enabled or disabled by default depending on the specific repository.
```

Comment:
If RH products' abilities are Satellite specific only, can we filter that sentence by `ifdef`?

Reply:
You can sync Red Hat repositories upstream as well.

Reply:
Alright :+1: